src/screens/CandidateDetail/TabBar/NewsTab.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {View, Text, Image, StyleSheet } from 'react-native';
+import { Icon } from 'react-native-elements'
+import Colors from '../../../styles/colors';
+import Mixins from '../../../styles/mixins';
+
+const NewsTab = (props) => {
+  return (
+    <View>
+      <View style={styles.newsCard}>
```

Comment:
It would be nice to have this inner section as its own component, since this `NewsTab` will eventually just map an array of info into different `NewsCard`s

Reply:
Done :)